lib/modules/datasource/conda/index.ts

```diff
@@ -74,4 +109,129 @@ export class CondaDatasource extends Datasource {
 
     return result.releases.length ? result : null;
   }
+
+  private async getReleasesFromPrefixDev(
+    channel: string,
+    packageName: string,
+  ): Promise<ReleaseResult | null> {
+    logger.debug(
+      { channel, packageName },
+      'lookup package from prefix.dev graphql API',
+    );
+
+    const versions = await this.getPrefixPagedResponse(
+      `
+  query search($channel: String!, $package: String!, $page: Int = 0) {
+    data: package(channelName: $channel, name: $package) {
+      data: versions(limit: 500, page: $page) {
+        pages
+        page {
+          version
+        }
+      }
+    }
+  }
+  `,
+      { channel, package: packageName },
+      prefixDevSchema.Version,
+    );
+
+    if (versions.length === 0) {
+      return null;
+    }
+
+    const files = await this.getPrefixPagedResponse(
+      `
+  query search($channel: String!, $package: String!, $page: Int = 0) {
+    data: package(channelName: $channel, name: $package) {
+      data: variants(limit: 500, page: $page) {
+        pages
+        page {
+          version
+          createdAt
+          yankedReason
+        }
+      }
+    }
+  }
+  `,
```

Comment:
Why are you not pulling this in the first API call?

Reply:
like the schema suggest, this is listing all files of a package, so then have differnet total page count with first api call

Reply:
It's possible to merge these 2 queries but I don't want to spend that mental energy to do it  😅

I think current implementation it much more easy to understand, let's refactor it when it cause trouble.